source/ui/BioXAS/BioXASControlEditor.cpp

```diff
@@ -262,6 +291,32 @@ void BioXASControlEditor::updateUnits()
 		BioXASValueEditor::setUnits(control_->units());
 }
 
+void BioXASControlEditor::updateProgressValueMinimum()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValueMinimum(control_->value());
+}
+
+void BioXASControlEditor::updateProgressValueMaximum()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValueMaximum(control_->setpoint());
+}
+
+void BioXASControlEditor::updateProgressValue()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValue(control_->value());
```

Comment:
There's no real need to scope it here. But the majority of methods are reimplemented between BioXASControlEditor and BioXASValueEditor. In this case, I think, it helps to be specific.